cmd/create.go

```diff
@@ -26,7 +26,8 @@ import (
 
 // Variables
 var (
-	CreateFiles []string
+	CreateNamespace string
```

Comment:
Same for `CreateFiles`, `ApplyFiles` and `DeleteFiles`, right?
Any ideas why was this set up like this in the first place?